dist/common/src/main/resources/packages/org.jboss.prospero/content/bin/prospero.bat

```diff
@@ -198,6 +198,13 @@ if exist "%PROSPERO_HOME%\jboss-modules.jar" (
   goto END
 )
 
+rem Use a copy of jboss-modules to avoid locking issues when jboss-modules is updated
+:COPY_JBOSS_MODULES
+    set "TMP_JBOSS_MODULES=%PROSPERO_HOME%\jboss-modules~%RANDOM%.tmp"
+    rem loop while we find a non-existing filename
+    if exist "%TMP_JBOSS_MODULES%" goto :COPY_JBOSS_MODULES
+    cp "%RUNJAR%" "%TMP_JBOSS_MODULES%"
```

Comment:
@spyrkob I found this, `cp` is not a valid command on Windows, this should be `copy` instead